src/style.css

```diff
@@ -1,3 +1,95 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  height: 60px;
+}
+
+a {
+  display: block;
+  color: #000;
+  text-decoration: none;
+  height: 60px;
+}
+
+.nav-logo a {
+  display: flex;
+  align-items: center;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  height: 60px;
+}
+
+.nav-item {
+  margin-left: 20px;
+  text-align: center;
+  height: 60px;
+  line-height: 60px;
+}
```

Comment:
It seems there might be a typo in the class name `.nav-item`. According to the checklist, class names should represent the meaning of the content. Based on your existing classes like `.nav__list`, it seems you're using BEM methodology, so this class should be `.nav__item` for consistency.